packages/@react-stately/virtualizer/src/Virtualizer.ts

```diff
@@ -114,8 +114,8 @@ export class Virtualizer<T extends object, V> {
   }
 
   private _renderView(reusableView: ReusableView<T, V>) {
-    let {type, key} = reusableView.layoutInfo;
-    reusableView.content = this.collection.getItem(key);
+    let {type, key, content} = reusableView.layoutInfo;
```

Comment:
I don't quite follow the implications of this change

Reply:
yep. this enables the layout to produce items that aren't in the original collection.